src/components/Directory/MentorCard/index.js

```diff
@@ -0,0 +1,120 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import useStyles from './MentorCard.styles';
+import {
+  Card,
+  CardHeader,
+  Divider,
+  Avatar,
+  Button,
+  CardContent,
+  Typography,
+  useMediaQuery,
+} from '@material-ui/core/';
+
+const MentorCard = ({
+  firstName,
+  lastName,
+  title,
+  mentorSkills,
+  bio,
+  initials,
+  onTestClick,
+}) => {
+  const isMobile = useMediaQuery('(max-width:767px)');
+  const classes = useStyles({ isMobile });
+  const {
+    root,
+    avatar,
+    booking,
+    skillList,
+    jobField,
+    mentorNameField,
+    bioSection,
+  } = classes;
+
+  const handleClick = e => {
+    e.preventDefault();
+    if (onTestClick) return onTestClick();
+    console.log('BOOKING FOR THIS MENTOR OPENS UP');
```

Comment:
let's remove the console.log and replace it with // TODO: Booking for this mentor opens up

Reply:
Resolved in commit:  a6ddf95
* Changed console to a comment that reads for next stage